superset/assets/stylesheets/superset.css

```diff
@@ -208,4 +208,8 @@ div.widget .slice_container {
 
 .table-condensed input[type="checkbox"] {
     float: left;
+}
+
+.form-inline {
```

Comment:
This will affect the entire theme for the whole app (SQL Lab). The selector should be more specific and target the component its in.